advanced/src/app/core/authentication/authentication.service.ts

```diff
@@ -2,6 +2,7 @@ import { Injectable } from '@angular/core';
 import { Observable, of } from 'rxjs';
```

Comment:
I guess this entire file could be updated like [as here](https://github.com/Cynthesize/cynthesize-frontend/blob/development/src/app/core/authentication/authentication.service.ts)

Reply:
That looks good!  The boilerplate won't need the `handleUserDatabaseEntry` stuff given it's handled by Auth0 rules, but otherwise pretty close to a drop-in replacement.  I'll try it out and update this PR with a new `authentication.service.ts`